### ISSUE TITLE
Tracing `cursor.execute(sql)` call (without params) stack traces

### ISSUE THREAD

**Issue by jordiecometrica** [opbeat/opbeat_python#135]
Here is an example of the stack trace:

```
File "/home/jordi/vcs/lol/local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  149.                     response = self.process_exception_by_middleware(e, request)

File "/home/jordi/vcs/lol/local/lib/python2.7/site-packages/django/core/handlers/base.py" in get_response
  147.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/home/jordi/vcs/lol/local/lib/python2.7/site-packages/django/utils/decorators.py" in inner
  184.                     return func(*args, **kwargs)

File "/home/jordi/vcs/lol/local/lib/python2.7/site-packages/django/utils/decorators.py" in _wrapped_view
  149.                     response = view_func(request, *args, **kwargs)

File "/home/jordi/vcs/lol/local/lib/python2.7/site-packages/django/views/decorators/cache.py" in _wrapped_view_func
  57.         response = view_func(request, *args, **kwargs)

File "/home/jordi/vcs/lol/local/lib/python2.7/site-packages/django/contrib/admin/sites.py" in inner
  244.             return view(request, *args, **kwargs)

File "/home/jordi/vcs/lol/local/lib/python2.7/site-packages/django/views/generic/base.py" in view
  68.             return self.dispatch(request, *args, **kwargs)

File "/home/jordi/vcs/lol/local/lib/python2.7/site-packages/django/views/generic/base.py" in dispatch
  88.         return handler(request, *args, **kwargs)

File "/home/jordi/vcs/lol/local/lib/python2.7/site-packages/django_object_actions/utils.py" in get
  240.         ret = view(request, *self.view_args)

File "/home/jordi/vcs/lol/local/lib/python2.7/site-packages/django_object_actions/utils.py" in decorated_function
  297.         return func(self, request, queryset)

File "/home/jordi/vcs/lol/lolomgwtf/clients/admin.py" in create_caravel_analytics
  70.             company.execute_caravel_analytics_sql()

File "/home/jordi/vcs/lol/lolomgwtf/clients/models.py" in execute_caravel_analytics_sql
  258.             cursor.execute(sql)

File "/home/jordi/vcs/lol/local/lib/python2.7/site-packages/django/db/backends/utils.py" in execute
  62.                 return self.cursor.execute(sql)

File "/home/jordi/vcs/lol/local/lib/python2.7/site-packages/opbeat/instrumentation/packages/dbapi2.py" in execute
  169.         return self._trace_sql(self.__wrapped__.execute, sql, params)

File "/home/jordi/vcs/lol/local/lib/python2.7/site-packages/opbeat/instrumentation/packages/dbapi2.py" in _trace_sql
  179.             return method(sql, params)

Exception Type: IndexError at /admin/clients/company/1048/actions/create_caravel_analytics/
Exception Value: tuple index out of range
```

I have a fix in progress, shouldn't be too hard.

**Comment by piquadrat** [comment]
@jordiecometrica sorry for the slowness on this. I'm trying to come up with a test case that triggers this error but am having a hard time. Would you mind posting the SQL query (minus any sensitive values of course) here?

**Comment by piquadrat** [comment]
Never mind, found one :)

**Comment by piquadrat** [comment]
Closed by #136
